### PR TITLE
- Optimization in CCClippingNode.

### PR DESCRIPTION
**Pull request by Teivaz**
Disable visiting when no objects has to be drawn under stencil.

### CODE REVIEW

**Comment by Teivaz** [comment]
Doesn't seem the build failure is caused by my changes.

File: cocos/2d/CCClippingNode.h [hunk 1]
```diff
@@ -58,7 +58,14 @@ class CC_DLL ClippingNode : public Node
...
     Node* getStencil() const;
     void setStencil(Node *stencil);
-    
+
+    /** If stencil has no childre it will not be drawn.
+    If you have custom stencil-based node with stencil drawing mechanics other then children-based,
+    then this method should return true every time you wish stencil to be visited.
+    By default returns true if has any children attached.
+    */
+    virtual bool hasContent();
```
Comment (minggo) [inline cocos/2d/CCClippingNode.h]: virtual bool hasContent() -> virtual bool hasContent() const

**Comment by Teivaz** [comment]
@minggo I've thought about this. Here's updated code.

**Comment by minggo** [comment]
Thanks.

**Pull request merged**